{
  "uid" : "b6ef3f7b5a6b0615",
  "name" : "test_contracts[input_data1]",
  "fullName" : "tests.openzeppelin.test_openzeppelin#test_contracts",
  "historyId" : "0978f566b0050886014c8f6266a2ba6f",
  "time" : {
    "start" : 1631802944331,
    "stop" : 1631802944331,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'",
  "statusTrace" : "@pytest.fixture(autouse=True)\n    def prepare_truffle_config():\n        preset_variables()\n        print(get_contracts_path())\n        print(get_migrations_path())\n        print(get_test_path())\n        clean_up_folder(get_contracts_path())\n>       clean_up_folder(get_migrations_path())\n\ntests/openzeppelin/test_openzeppelin.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nfolder = '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'\n\n    def clean_up_folder(folder: str):\n>       for filename in os.listdir(folder):\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'\n\nsrc/helpers/shell/file_system.py:6: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "prepare_truffle_config",
    "time" : {
      "start" : 1631802944332,
      "stop" : 1631802944340,
      "duration" : 8
    },
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'\n",
    "statusTrace" : "  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/home/runner/.local/share/virtualenvs/neon-compatibility-g882AfFF/lib/python3.8/site-packages/_pytest/fixtures.py\", line 925, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/tests/openzeppelin/test_openzeppelin.py\", line 63, in prepare_truffle_config\n    clean_up_folder(get_migrations_path())\n  File \"/home/runner/work/neon-compatibility/neon-compatibility/src/helpers/shell/file_system.py\", line 6, in clean_up_folder\n    for filename in os.listdir(folder):\n",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  } ],
  "testStage" : {
    "status" : "broken",
    "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'",
    "statusTrace" : "@pytest.fixture(autouse=True)\n    def prepare_truffle_config():\n        preset_variables()\n        print(get_contracts_path())\n        print(get_migrations_path())\n        print(get_test_path())\n        clean_up_folder(get_contracts_path())\n>       clean_up_folder(get_migrations_path())\n\ntests/openzeppelin/test_openzeppelin.py:63: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nfolder = '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'\n\n    def clean_up_folder(folder: str):\n>       for filename in os.listdir(folder):\nE       FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/migrations/'\n\nsrc/helpers/shell/file_system.py:6: FileNotFoundError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "2c2ae81c2905fcd5",
      "name" : "stdout",
      "source" : "2c2ae81c2905fcd5.txt",
      "type" : "text/plain",
      "size" : 236
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "tests.openzeppelin"
  }, {
    "name" : "suite",
    "value" : "test_openzeppelin"
  }, {
    "name" : "host",
    "value" : "fv-az39-805"
  }, {
    "name" : "thread",
    "value" : "3949-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.openzeppelin.test_openzeppelin"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "input_data",
    "value" : "InputData(test_file='access/Ownable.test.js', contracts=['ctr', 'ctr2'], migration_script='script1111')"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 0,
        "unknown" : 0,
        "total" : 2
      },
      "items" : [ {
        "uid" : "19c9ff428c3a905b",
        "reportUrl" : "https://neonlabsorg.github.io/neon-compatibility/75//#testresult/19c9ff428c3a905b",
        "status" : "broken",
        "statusDetails" : "FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/neon-compatibility/neon-compatibility/OpenZeppelin/contracts/'",
        "time" : {
          "start" : 1631802267922,
          "stop" : 1631802267922,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "b6ef3f7b5a6b0615.json",
  "parameterValues" : [ "InputData(test_file='access/Ownable.test.js', contracts=['ctr', 'ctr2'], migration_script='script1111')" ]
}